react-native-libraries.json

```diff
@@ -10881,5 +10881,13 @@
       "https://raw.githubusercontent.com/callstack/react-native-windows-hello/main/.github/Resources/RNW-Hello-FullExample.gif"
     ],
     "windows": true
+  },
+  {
+    "githubUrl": "https://github.com/tushu4212/rntemplate",
+    "npmPkg": "@tushar4212/rntemplate",
+    "nameOverride": "rntemplate",
```

Comment:
Hello @tushu4212, thank you for adding new entry to the directory! 👍 

One suggestion before shipping, let's stick to the full template/package name to avoid users confusion:

```suggestion
```